src/types/commits.ts

```diff
@@ -17,3 +17,5 @@ export enum ChartTypes {
   Absolute = "absolute",
   Percentage = "percentage",
 }
+
+export const commitQueueRequester = "merge_test";
```

Comment:
we have a constants directory. I think this should live in one of the files there.